Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Responses.Commands;
+using Watchman.DomainModel.Responses.Queries;
+
+namespace Watchman.Discord.Areas.Responses.Controllers
+{
+    public class ResponsesController : IController
+    {
+        private readonly IQueryBus _queryBus;
+        private readonly ICommandBus _commandBus;
+        private readonly UsersService _usersService;
+        private readonly DirectMessagesService _directMessagesService;
+        private readonly MessagesServiceFactory _messagesServiceFactory;
+        private readonly UsersRolesService _usersRolesService;
+        private readonly Services.ResponsesService _responsesService;
+
+        public ResponsesController(IQueryBus queryBus, ICommandBus commandBus, UsersService usersService, DirectMessagesService directMessagesService, MessagesServiceFactory messagesServiceFactory, UsersRolesService usersRolesService, Services.ResponsesService responsesService)
+        {
+            this._queryBus = queryBus;
+            this._commandBus = commandBus;
+            this._usersService = usersService;
+            this._directMessagesService = directMessagesService;
+            this._messagesServiceFactory = messagesServiceFactory;
+            this._responsesService = responsesService;
+            _usersRolesService = usersRolesService;
+        }
+
+        [AdminCommand]
+        [DiscordCommand("add response")]
+        public async Task AddResponse(DiscordRequest request, Contexts contexts)
```

Comment:
Jeszcze spróbuj zmienić DiscordRequest na komende z IBotCommand
Wtedy byś nie musiał pisać 

```csharp
            var onEvent = request.Arguments.FirstOrDefault(x => x.Name?.ToLowerInvariant() == "onevent")?.Value;
            var message = request.Arguments.FirstOrDefault(x => x.Name?.ToLowerInvariant() == "message")?.Value;
```

ale jeśli czujesz że wtedy komenda wyglądałaby gorzej, bo tamto jeszcze nie jest dopracowane, to zostaw jak jest

Przykładowo byłaby komenda 
AddResponseCommand 
czyli użycie

-addresponse -onevent TEST -message "test"

powinny być dwa property

[SingleWord]
public string OnEvent {get; set;} 
[Text]
public string Message {get;set;}

zobacz sobie jak to wygląda w UselesFeatures

Reply:
zostawiam jak jest